### ISSUE TITLE
grunt serve:dist not building all files due to misconfigured compass?

### ISSUE THREAD

**Issue by grctest** [oxarbitrage/open-explorer#9]
During the `grunt serve:dist --force` command, the following errors are observed:

```
    Execution Time (2018-02-10 01:34:31 UTC)
    compass:dist  1.8s  100%
    Total 1.8sCompass can't find any Sass files to compile.
    Is your compass configuration correct?.
    If you're trying to start a new project, you have left off the directory argument.
    Run "compass -h" to get help.
```

Perhaps not errors:
```
Running "cssmin:dist" (cssmin) task
>> No files created.

[ ...... ]

Running "uglify:dist" (uglify) task
>> No files created.
```

Within the dist folder noticeable issues:
* index.html file not brought into the dist folder -> copied file manually
* boostrap css not included in the bower_components folder.

Most of the javascript files didn't generate (the html files are however present):
```
Loading failed for the <script> with source “http://hertz.network/bower_components/jquery/dist/jquery.min.js”.
index.html:53
Loading failed for the <script> with source “http://hertz.network/bower_components/angular/angular.min.js”.
index.html:54
Loading failed for the <script> with source “http://hertz.network/bower_components/bootstrap/dist/js/bootstrap.min.js”.
index.html:55
Loading failed for the <script> with source “http://hertz.network/bower_components/angular-websocket/dist/angular-websocket.min.js”.
index.html:56
Loading failed for the <script> with source “http://hertz.network/bower_components/angular-route/angular-route.min.js”.
index.html:57
Loading failed for the <script> with source “http://hertz.network/bower_components/angular-aria/angular-aria.min.js”.
index.html:58
Loading failed for the <script> with source “http://hertz.network/bower_components/angular-animate/angular-animate.min.js”.
index.html:59
Loading failed for the <script> with source “http://hertz.network/bower_components/angular-translate/angular-translate.min.js”.
index.html:60
Loading failed for the <script> with source “http://hertz.network/bower_components/angular-translate-loader-static-files/angular-translate-loader-static-files.min.js”.
index.html:61
Loading failed for the <script> with source “http://hertz.network/bower_components/angular-google-analytics/dist/angular-google-analytics.js”.
index.html:62
Loading failed for the <script> with source “http://hertz.network/bower_components/angular-bootstrap/ui-bootstrap-tpls.min.js”.
index.html:66
Loading failed for the <script> with source “http://hertz.network/bower_components/echarts/dist/echarts.min.js”.
index.html:67
Loading failed for the <script> with source “http://hertz.network/bower_components/ngECharts/dist/ngecharts.js”.
index.html:68
Loading failed for the <script> with source “http://hertz.network/bower_components/angular-loading-bar/build/loading-bar.min.js”.
index.html:69
Loading failed for the <script> with source “http://hertz.network/bower_components/jdenticon/dist/jdenticon.min.js”.
index.html:73
Loading failed for the <script> with source “http://hertz.network/bower_components/js-sha256/build/sha256.min.js”.
index.html:74
Loading failed for the <script> with source “http://hertz.network/app.module.js”.
index.html:83
Loading failed for the <script> with source “http://hertz.network/sections/layout/nav.module.js”.
index.html:84
Loading failed for the <script> with source “http://hertz.network/sections/accounts/accounts.module.js”.
index.html:85
Loading failed for the <script> with source “http://hertz.network/sections/assets/assets.module.js”.
index.html:86
Loading failed for the <script> with source “http://hertz.network/sections/markets/markets.module.js”.
index.html:87
Loading failed for the <script> with source “http://hertz.network/sections/committee_members/committee_members.module.js”.
index.html:88
Loading failed for the <script> with source “http://hertz.network/sections/fees/fees.module.js”.
index.html:89
Loading failed for the <script> with source “http://hertz.network/sections/objects/objects.module.js”.
index.html:90
Loading failed for the <script> with source “http://hertz.network/sections/proxies/proxies.module.js”.
index.html:91
Loading failed for the <script> with source “http://hertz.network/sections/search/search.module.js”.
index.html:92
Loading failed for the <script> with source “http://hertz.network/sections/txs/txs.module.js”.
index.html:93
Loading failed for the <script> with source “http://hertz.network/sections/voting/voting.module.js”.
index.html:94
Loading failed for the <script> with source “http://hertz.network/sections/witnesses/witnesses.module.js”.
index.html:95
Loading failed for the <script> with source “http://hertz.network/sections/workers/workers.module.js”.
index.html:96
Loading failed for the <script> with source “http://hertz.network/sections/operations/operations.module.js”.
index.html:97
Loading failed for the <script> with source “http://hertz.network/sections/blocks/blocks.module.js”.
index.html:98
Loading failed for the <script> with source “http://hertz.network/chart/chart.module.js”.
index.html:99
Loading failed for the <script> with source “http://hertz.network/core/app.config.js”.
index.html:100
Loading failed for the <script> with source “http://hertz.network/core/app.controller.js”.
index.html:101
Loading failed for the <script> with source “http://hertz.network/core/config.route.js”.
index.html:102
Loading failed for the <script> with source “http://hertz.network/core/i18n.js”.
index.html:103
Loading failed for the <script> with source “http://hertz.network/sections/dashboard/dashboard.controller.js”.
index.html:104
Loading failed for the <script> with source “http://hertz.network/sections/layout/header.controller.js”.
index.html:105
Loading failed for the <script> with source “http://hertz.network/sections/layout/nav.directive.js”.
index.html:106
Loading failed for the <script> with source “http://hertz.network/sections/accounts/accounts.controller.js”.
index.html:107
Loading failed for the <script> with source “http://hertz.network/sections/assets/assets.controller.js”.
index.html:108
Loading failed for the <script> with source “http://hertz.network/sections/markets/markets.controller.js”.
index.html:109
Loading failed for the <script> with source “http://hertz.network/sections/committee_members/committee_members.controller.js”.
index.html:110
Loading failed for the <script> with source “http://hertz.network/sections/fees/fees.controller.js”.
index.html:111
Loading failed for the <script> with source “http://hertz.network/sections/objects/objects.controller.js”.
index.html:112
Loading failed for the <script> with source “http://hertz.network/sections/proxies/proxies.controller.js”.
index.html:113
Loading failed for the <script> with source “http://hertz.network/sections/search/search.controller.js”.
index.html:114
Loading failed for the <script> with source “http://hertz.network/sections/txs/txs.controller.js”.
index.html:115
Loading failed for the <script> with source “http://hertz.network/sections/voting/voting.controller.js”.
index.html:116
Loading failed for the <script> with source “http://hertz.network/sections/witnesses/witnesses.controller.js”.
index.html:117
Loading failed for the <script> with source “http://hertz.network/sections/workers/workers.controller.js”.
index.html:118
Loading failed for the <script> with source “http://hertz.network/sections/operations/operations.controller.js”.
index.html:119
Loading failed for the <script> with source “http://hertz.network/sections/blocks/blocks.controller.js”.
index.html:120
Loading failed for the <script> with source “http://hertz.network/chart/echarts.controller.js”.
index.html:121
```

---

TL;DR: Compass configuration missing caused the above?

Can I just copy all the missing files from the /app/ folder?

**Comment by oxarbitrage** [comment]
OK .I'll check that out tomorrow. I think I didn't tested the dist in the
final open source version. Thanks.

**Comment by grctest** [comment]
Any luck @oxarbitrage or do you think it's my own server environment problem? Thanks 👍

**Comment by oxarbitrage** [comment]
hey, sorry the delay. 

no, it is not your server, i am seeing the same problem, something wrong with some of the last changes i made, trying to find out where. was not able to find it a few days ago, i am giving it another try today. ill update you as soon ass i get this working in a few hours i hope.

**Comment by oxarbitrage** [comment]
please use last version(`git clone https://github.com/oxarbitrage/open-explorer`) or apply changes to `Gruntfile.js` here: 
https://github.com/oxarbitrage/open-explorer/commit/778ccfe63a6f045e8d9f4f2842a3b8f1ae3dd08d

working for me now, index is copied, the rest was already working fine in here.

please check and let me know.

**Comment by grctest** [comment]
@oxarbitrage That's fixed it now, got it working - thanks 👍
